src/components/Results.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+
+class Results extends Component {
+    render() {
+        return (
+            <div>
+                <table className="table search-table">
+                    <thead>
+                        <tr>
+                            <th>Search Results</th>
+                            <th />
+                            <th></th>
+                            <th />
+                        </tr>
+                        <tr>
+                            <th>Title</th>
```

Comment:
This works, but can you see how we could improve the readability a bit by extracting some more components? Maybe a `TableHeader`? There's at least 1 more component that might be extracted here.